sympy/solvers/solveset.py

```diff
@@ -192,6 +191,11 @@ def inv(trig):
                 invs += Union(*[imageset(Lambda(n, L(g)), S.Integers) for g in g_ys])
             return _invert_real(f.args[0], invs, symbol)
 
+    if (isinstance(a,log) for a in f.args):
```

Comment:
For this [testcase](https://github.com/sympy/sympy/pull/10733/files#diff-85baa04bbf4e1dfd9128782738e45424R240)